swift/ql/test/library-tests/dataflow/taint/libraries/TaintInline.expected

```diff
@@ -1,2 +1,13 @@
-failures
 testFailures
+| optionset.swift:60:49:61:1 | // $ tainted=60\n | Missing result: tainted=60 |
+| optionset.swift:65:58:66:1 | // $ tainted=65\n | Missing result: tainted=65 |
+| string.swift:237:35:238:1 | // $ tainted=217\n | Missing result: tainted=217 |
+| string.swift:244:33:245:1 | // $ tainted=217\n | Missing result: tainted=217 |
```

Comment:
Line 237 is this:
```
sink(arg: tainted.dropLast(10)) // $ tainted=217
```
Line 244 is this:
```
sink(arg: tainted.reversed()) // $ tainted=217
```
I'll look into whether the library declarations of some of these functions has changed so we're no longer matching them???

Reply:
It looks like some of the models (including these two) need updating because some things are defined in slightly different places in Swift 6.  At least that's my understanding.  I'm working on a PR to this PR that should fix a lot of this.

Reply:
PR here: https://github.com/github/codeql/pull/17891

There's still an issue to do with conversions and/or enums that I haven't looked into.